examples/ts-node/src/index.ts

```diff
@@ -0,0 +1,42 @@
+import { SubspaceClient, Identity } from "subspace.js";
+import { mnemonicGenerate } from "@polkadot/util-crypto";
```

Comment:
I would just hardcode `//Alice` instead of using this. Also it is not in explicit dependencies.